build failed

/kind bug


**What happened**:
```
$ GOPATH=/pkgs/go make install
go build -o /pkgs/go/bin/ks -ldflags="-X main.version=dev-2018-12-17T13:35:57+0800 -X main.apimachineryVersion=kubernetes-1.10.4 -X generator.revision=7c76c9a36caccb5261c05c5152919b6ae1badcc0  "  ./cmd/ks
# github.com/ksonnet/ksonnet/vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/printer
vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/printer/printer.go:185:8: undefined: strings.Builder
vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/printer/printer.go:225:8: undefined: strings.Builder
vendor/github.com/ksonnet/ksonnet-lib/ksonnet-gen/printer/printer.go:748:11: undefined: strings.Builder
$  go version
go version go1.9.4 linux/amd64
```

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
follow [doc](https://github.com/ksonnet/ksonnet/blob/master/docs/build-install.md)

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):`Client Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:46:00Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"10", GitVersion:"v1.10.5", GitCommit:"32ac1c9073b132b8ba18aa830f46b77dcceb0723", GitTreeState:"clean", BuildDate:"2018-06-21T11:34:22Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
`

--- a/docs/build-install.md
+++ b/docs/build-install.md
@@ -28,3 +28,6 @@ If your ksonnet is properly installed, you should be able to run `ks --help` and
 
 * **If your error is "command not found", make sure that Go binaries are included in your $PATH**.
   You can do this by running `PATH=$PATH:$GOPATH/bin`.
+
+* **You need to use go over 1.10 version**
+  Use `go version` to check it.

